scripts/script.js

```diff
@@ -0,0 +1,23 @@
+
+window.onload=function(){
+    document.getElementById("menu-hamburger").addEventListener("click", toggleMenu);
+    window.addEventListener("resize", function(event) {
+        if (navBar.classList.contains("nav-mobile")) {
+            navBar.classList.remove("nav-mobile");
+            navBar.classList.add("navigation");
+        }
+    })
+}
+
+const toggleMenu = () => {
+    var navBar=document.getElementById("navigation");
+    console.log("clicked")
+    if (navBar.classList.contains("nav-mobile")) {
+        navBar.classList.remove("nav-mobile");
+        navBar.classList.add("navigation");
+    } else {
+        navBar.classList.remove("navigation");
+        navBar.classList.add("nav-mobile");
+    }
+   
```

Comment:
Nice, now you want to rearrange your code so that `navBar ` is defined in the resize function, for example like this:

```suggestion
    var navBar = document.getElementById("navigation");
    window.addEventListener("resize", function (event) {
        if (navBar.classList.contains("nav-mobile")) {
            navBar.classList.remove("nav-mobile");
            navBar.classList.add("navigation");
        }
    })

    const toggleMenu = () => {
        console.log("clicked")
        if (navBar.classList.contains("nav-mobile")) {
            navBar.classList.remove("nav-mobile");
            navBar.classList.add("navigation");
        } else {
            navBar.classList.remove("navigation");
            navBar.classList.add("nav-mobile");
        }
    }
    document.getElementById("menu-hamburger").addEventListener("click", toggleMenu);   
```